code/modules/admin/verbs/listahelps.dm

```diff
@@ -0,0 +1,318 @@
+/datum/adminticket
+	var/ID = ""
+	var/user = ""
+	var/uckey
+	var/admin = "N/A"
+	var/msg = ""
+	var/resolved = "No"
+	var/permckey = ""
+	var/permuser = ""
+	var/uID = ""
+	var/active = "No"
+	var/logs = list()
+	var/replying = 0
+	var/mob
+
+client/proc/list_ahelps(var/user, var/resolved)
```

Comment:
also you don't need to put var/ in the arguments,and you may want to typecast the user as a mob perhaps